modules/ocf_desktop/files/xsession/fix-displays

```diff
@@ -54,11 +54,26 @@ if __name__ == '__main__':
     else:
         sorted_monitors = list(sorted(monitors))
 
-    prev = None
-    for monitor in sorted_monitors:
-        if prev:
-            subprocess.check_call(['xrandr', '--output', monitor, '--right-of', prev])
-        prev = monitor
+    # special case for eruption: right monitor is rotated left 90 degrees
+    if len(monitors) == 3 and hostname == 'eruption':
```

Comment:
I'm not a fan of encoding special cases in the program like this. Is there a way to encode this info per host that the program could fetch? or even some kind of configuration file that overrides specifically for eruption.

Reply:
Could be worth looking into something like [autorandr](https://github.com/phillipberndt/autorandr) (which is in the Debian Buster repos) to dynamically set the monitor configs.

Reply:
autorandr looks nice